direct/io-bulkfs/src/main/java/cz/o2/proxima/direct/bulk/FileFormatUtils.java

```diff
@@ -97,6 +97,10 @@ static FileFormat getFileFormatFromName(String format, boolean gzip) {
     if ("json".equals(format)) {
       return FileFormat.json(gzip);
     }
+    if ("parquet".equals(format)) {
+      // just an shortcut for parquet file format
+      format = "cz.o2.proxima.direct.bulk.fs.parquet.ParquetFileFormat";
```

Comment:
```suggestion
      return new ParquetFileFormat();
```

Why don't you instantiate the format immediately?

Reply:
We can't do that because it will create circular dependency between `proxima-direct-io-bulkfs` and `proxima-direct-io-bulkfs-parquet`